CodePushPackage.m

```diff
@@ -228,8 +228,16 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
             NSString * unzippedFolderPath = [CodePushPackage getUnzippedFolderPath];
             NSMutableDictionary * mutableUpdatePackage = [updatePackage mutableCopy];
             if (isZip) {
-                NSError *nonFailingError = nil;
+                if ([[NSFileManager defaultManager] fileExistsAtPath:unzippedFolderPath]) {
```

Comment:
Added the comment. Also [CodePushPackage clearUpdates] deletes the entire CodePush folder which contains everything that we use including this unzip folder destination path.